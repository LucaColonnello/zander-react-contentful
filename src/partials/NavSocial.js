import React from 'react'

const NavSocial = () => {
  return (
    <nav className="navSocial">
      <a href="http://twitter.com/mrmartineau" target="_blank" title="Follow Zander on Twitter" className="navSocial-link">
        <svg className="navSocial-link-icon">
          <use xlinkHref="#icon-twitter"></use>
        </svg>
      </a>

      <a href="http://github.com/mrmartineau" target="_blank" title="View Zander's open source code on Github" className="navSocial-link">
        <svg className="navSocial-link-icon">
          <use xlinkHref="#icon-github"></use>
        </svg>
        <span className="is-visuallyHidden">Github</span>
      </a>

      <a href="https://medium.com/@mrmartineau/" target="_blank" title="View Zander's Medium.com profile" className="navSocial-link">
        <svg className="navSocial-link-icon">
          <use xlinkHref="#icon-medium"></use>
        </svg>
      </a>

      <a href="https://instagram.com/mrmartineau" target="_blank" title="View Zander's npm packages" className="navSocial-link">
        <svg className="navSocial-link-icon">
          <use xlinkHref="#icon-instagram"></use>
        </svg>
      </a>

      <a href="https://www.npmjs.com/~mrmartineau" target="_blank" title="View Zander's npm packages" className="navSocial-link">
        <svg className="navSocial-link-icon">
          <use xlinkHref="#icon-npm"></use>
        </svg>
      </a>
    </nav>
  )
}

export default NavSocial
