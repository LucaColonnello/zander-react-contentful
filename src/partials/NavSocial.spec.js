import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import NavSocial from './NavSocial';

test('NavSocial snapshot test', () => {
  const component = shallow(<NavSocial />);
  const tree = shallowToJson(component);
  expect(tree).toMatchSnapshot();
});
