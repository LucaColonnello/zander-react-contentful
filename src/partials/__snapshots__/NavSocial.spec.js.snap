// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavSocial snapshot test 1`] = `
<nav
  className="navSocial"
>
  <a
    className="navSocial-link"
    href="http://twitter.com/mrmartineau"
    target="_blank"
    title="Follow Zander on Twitter"
  >
    <svg
      className="navSocial-link-icon"
    >
      <use
        xlinkHref="#icon-twitter"
      />
    </svg>
  </a>
  <a
    className="navSocial-link"
    href="http://github.com/mrmartineau"
    target="_blank"
    title="View Zander's open source code on Github"
  >
    <svg
      className="navSocial-link-icon"
    >
      <use
        xlinkHref="#icon-github"
      />
    </svg>
    <span
      className="is-visuallyHidden"
    >
      Github
    </span>
  </a>
  <a
    className="navSocial-link"
    href="https://medium.com/@mrmartineau/"
    target="_blank"
    title="View Zander's Medium.com profile"
  >
    <svg
      className="navSocial-link-icon"
    >
      <use
        xlinkHref="#icon-medium"
      />
    </svg>
  </a>
  <a
    className="navSocial-link"
    href="https://instagram.com/mrmartineau"
    target="_blank"
    title="View Zander's npm packages"
  >
    <svg
      className="navSocial-link-icon"
    >
      <use
        xlinkHref="#icon-instagram"
      />
    </svg>
  </a>
  <a
    className="navSocial-link"
    href="https://www.npmjs.com/~mrmartineau"
    target="_blank"
    title="View Zander's npm packages"
  >
    <svg
      className="navSocial-link-icon"
    >
      <use
        xlinkHref="#icon-npm"
      />
    </svg>
  </a>
</nav>
`;
